cluster/helpers.go

```diff
@@ -42,6 +42,13 @@ const (
 	k1SigLen = 65
 	// k1RecIdx is the secp256k1 signature recovery id index.
 	k1RecIdx = 64
+	// Fork version of sepolia testnet.
+	sepoliaForkVersion = "0x90000069"
```

Comment:
go comments should have the variable name as the prefix: `// sepoliaForkVersion is the Sepolia testnet fork version. `

Reply:
move this to a test file if only used in testing.